6 - Functional enrichment analysis.Rmd

```diff
@@ -57,6 +57,62 @@ length(FE_genes_entrez_unique)
 
 
 
+## Optional
+Performing functional enrichment on up- and down-regulated genes separately.
+In the event which the user wishes to perform functional enrichment on up- and down-regulated genes separately, this cell outputs the necessary variables for that.
+The unique Ensembl and Entrez IDs can be retrieve from the up_reg and down_reg dataframes individually and be piped into the downstream functional enrichment steps.
+`FE_genes_up_reg_ensembl_unique` and `FE_genes_up_reg_entrez_unique` would have the Ensembl and Entrez IDs for up-regulated genes respectively.
+`FE_genes_down_reg_ensembl_unique` and `FE_genes_down_reg_entrez_unique` would have the Ensembl and Entrez IDs for down-regulated genes respectively.
+```{r}
+FE_genes_up_reg_cpm_filtered <- RNA_seq_DE_cpm_filtered_annotated %>% dplyr::filter(
+  padj < padj_cutoff 
+  & log2FoldChange > 1
```

Comment:
Instead of using `log2FoldChange > 1`, it may be better to use `log2FoldChange > l2fc_cutoff`, which was set in the previous Rmds. Or perhaps specify the l2fc_cutoff argument again and use the variable instead of 1 here?